bin/update

```diff
@@ -28,18 +40,20 @@ chdir APP_ROOT do
   # If we need a "reset all the things" script, we should call it bin/reset.
   system! 'bundle update'
 
+  # load 'Rakefile'
+  load_rakefile
   puts "\n== Updating database =="
-  system! 'bin/rails db:migrate'
+  Rake::Task["db:migrate"].invoke
 
   puts "\n== Seeding database =="
-  system! "bin/rails db:seed GOOD_MIGRATIONS=skip"
+  Rake::Task["db:seed"].invoke
```

Comment:
Considering all factors, I'm 👎  on this for the reason stated here. We had really annoying issues with dirty rake tasks and it's a somewhat accepted behavior on the Rails end. I do not see the speed improvement as beneficial enough to cause future issues with that, as well as other considerations that you've already outlined very well in your Considerations section of your description.